projects/1.21-fabric/assets/archers-expansion/archers_expansion/lang/zh_cn.json

```
@@ -0,0 +1,97 @@
+    "item.archers_expansion.netherite_tundra_hunter_chest": "下界合金寒霜外套",
+    "item.archers_expansion.netherite_tundra_hunter_head": "下界合金寒霜兜帽",
+    "item.archers_expansion.netherite_tundra_hunter_legs": "下界合金寒霜护腿",
+    "item.archers_expansion.netherite_tundra_hunter_feet": "下界合金寒霜靴子",
+  
+    "spell.archers_expansion.arctic_volley.name": "寒冰箭雨",
+    "spell.archers_expansion.arctic_volley.description": "一次性射出8支寒冰箭，造成 {damage} 点伤害并冻结目标。",
+    "spell.archers_expansion.enchanted_crystal_arrow.name": "附魔水晶箭",
+    "spell.archers_expansion.enchanted_crystal_arrow.description": "造成 {damage} 点伤害，使目标眩晕并冻结。",
+    "spell.archers_expansion.frozen_pact.name": "冰霜契约",
+    "spell.archers_expansion.frozen_pact.description": "根据目标的冰霜层数造成伤害，并在 {effect_duration} 秒内降低其攻击力。",
+    "spell.archers_expansion.frozen_shot.name": "冰霜射击",
+    "spell.archers_expansion.frozen_shot.description": "你的接下来几次射击会为目标叠加冰霜效果，使其减速并逐渐冻结。",
+  
+    "effect.archers_expansion.enchanted_crystal_arrow": "水晶箭矢",
+    "effect.archers_expansion.enchanted_crystal_arrow.description": "定身目标并为其叠加冰霜层数。",
+    "effect.archers_expansion.frozen_pact": "冰霜契约",
+    "effect.archers_expansion.frozen_pact.description": "根据目标的冰霜层数造成伤害，并降低其攻击力。",
+    "effect.archers_expansion.frozen_shot": "冰霜射击",
+    "effect.archers_expansion.frozen_shot.description": "你的接下来几次射击会为目标叠加冰霜效果。",
+  
+    "item.archers_expansion.war_archer_spell_book": "战弓狩典",
+    "item.archers_expansion.war_archer.spell_scroll": "战弓卷轴",
+    "item.archers_expansion.war_archer_chest": "弓手胸甲",
+    "item.archers_expansion.war_archer_head": "弓手头盔",
+    "item.archers_expansion.war_archer_legs": "弓手护腿",
+    "item.archers_expansion.war_archer_feet": "弓手靴子",
+    "item.archers_expansion.netherite_war_archer_chest": "下界合金弓手胸甲",
+    "item.archers_expansion.netherite_war_archer_head": "下界合金弓手头盔",
+    "item.archers_expansion.netherite_war_archer_legs": "下界合金弓手护腿",
+    "item.archers_expansion.netherite_war_archer_feet": "下界合金弓手靴子",
+  
+    "spell.archers_expansion.dual_shot.name": "双重射击",
+    "spell.archers_expansion.dual_shot.description": "一次性射出两支箭",
+    "spell.archers_expansion.pin_down.name": "钉刺压制",
+    "spell.archers_expansion.pin_down.description": "造成 {damage} 点伤害，并使目标在 {effect_duration} 秒内无法移动或跳跃。",
+    "spell.archers_expansion.point_blank_shot.name": "贴脸射击",
+    "spell.archers_expansion.point_blank_shot.description": "造成 {damage} 点伤害，且目标离施法者越近，击退效果越强。",
+    "spell.archers_expansion.smoldering_arrow.name": "炽燃箭矢",
+    "spell.archers_expansion.smoldering_arrow.description": "你的下一次射击会在目标附近引发小范围爆炸，对周围生物造成伤害并点燃它们。",
+  
+    "effect.archers_expansion.smoldering_arrow": "炽燃箭矢",
+    "effect.archers_expansion.smoldering_arrow.description": "在目标附近引发小范围爆炸，对周围生物造成伤害并点燃它们。",
+    "effect.archers_expansion.smoldering_arrows": "炽燃箭矢",
+    "effect.archers_expansion.smoldering_arrows.description": "你的下一次射击会在目标附近引发小范围爆炸，对周围生物造成伤害并点燃它们。",
+    "effect.archers_expansion.point_blank_shot": "贴脸射击",

```

```suggestion
    "effect.archers_expansion.point_blank_shot": "抵近射击",

```
贴脸射击也没错了，但是用抵近射击还是更为规范化一点，改不改尊重您的意见。